content/en-us/reference/engine/classes/UnreliableRemoteEvent.yaml

```diff
@@ -12,14 +41,33 @@ deprecation_message: ''
 properties: []
 methods:
   - name: UnreliableRemoteEvent:FireAllClients
-    summary: ''
-    description: ''
+    summary: |
+      Fires the `Class.UnreliableRemoteEvent.OnClientEvent|OnClientEvent` event for each
+      client connected to the same `Class.UnreliableRemoteEvent`. Has a 900 byte limit to the payload of the event, otherwise event is dropped.
```

Comment:
```suggestion
      Fires the `Class.UnreliableRemoteEvent.OnClientEvent|OnClientEvent` event for each
      client connected to the same `Class.UnreliableRemoteEvent`. Has a 900 byte limit to
      the payload of the event, otherwise event is dropped.
```